scripts/migrate.sh

```
@@ -0,0 +1,61 @@
+    local migration_file=$1
+    local migration_name=$(basename "$migration_file" .sql)
+
+    echo "📝 Applying $migration_name via Supabase MCP..."
+
+    # Use the Supabase MCP tool to apply the migration
+    # This bypasses CLI connectivity issues
+    doppler run -- pnpm exec supabase-mcp apply-migration "$migration_file"
+}
+
+# Check for unapplied migrations
+UNAPPLIED=$(ls -1 supabase/migrations/*.sql 2>/dev/null | wc -l)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)